sorc/chgres_cube.fd/input_data.F90

```diff
@@ -49,6 +54,8 @@ module input_data
                                     longitude_input_grid, &
                                     inv_file
 
+ use surface
```

Comment:
There's a circular dependency between `surface.F90` and `input_data.F90` that causes the CMake build to fail for me.

Reply:
@kgerheiser Thanks for pointing that out. Looks like that line got left hanging there on accident. I'll remove it.